tap_facebook/__init__.py

```diff
@@ -169,6 +169,11 @@ def should_retry_api_error(exception):
         **wait_gen_kwargs
     )
 
+@retry_pattern(backoff.constant, FacebookRequestError, max_tries=5, interval=1)
+def api_get_with_retry(job):
+    job = job.api_get()
+    return job
+
```

Comment:
I'd move this closer to usage inside the class and mark it "private" in the python sense by prefixing with `__`.

Pylint will probably warn about no self usage, but I think you can use `@staticmethod` to stop that. So like:

```
@staticmethod
@retry_pattern(backoff.constant, FacebookRequestError, max_tries=5, interval=1)
def __api_get_with_retry(job):
    job = job.api_get()
    return job
```
Then the usage would be
```
AdsInsights.__api_get_with_retry(job)
```

Reply:
The benefit of this is that it makes it clear that this is only for insights usage and to keep it localized for if we want to refactor the class into its own module.